packages/react-frontend/src/components/TableTask.jsx

```diff
@@ -29,6 +40,19 @@ function TableTask(props) {
                 <Tag className="date-tag" icon="pi pi-calendar">
                     {new Date(task.due_date_time).toLocaleDateString("en-US")}
                 </Tag>
+                <button className="delete-btn" onClick={() => setEditDialogVisible(true)}>
```

Comment:
This code is duplicates our edit and remove buttons. The div just below handles edit and remove, these buttons are redundant

Reply:
Are you proposing removing code between lines 43 and 55? I don't see any duplicate buttons, those lines of code should create one copy of the edit button, one copy of the share button, and one copy of the delete button

Reply:
Never mind, I see the problem now. The addition of the icons for edit and delete was not present on my dev branch. Will fix and update